lib/logflare/backends/adaptor/clickhouse_adaptor.ex

```diff
@@ -0,0 +1,163 @@
+defmodule Logflare.Backends.Adaptor.ClickhouseAdaptor do
+  @moduledoc """
+  ClickHouse backend adaptor
+
+  ### Table Creation
+
+  Currently this adaptor expects a table to be created with the following schema.
+
+  Adjust the database / table names as needed.
+
+  ```sql
+  CREATE TABLE "default"."supabase_log_ingress" (
+        "id" UUID,
+        "event_message" String,
+        "body" String,
```

Comment:
Lets change this to `metadata` and just go ahead and make it a json field.

This would let us play with how efficient json querying is while we load test the ingest side of things.

Reply:
Hmm, perhaps nevermind:

```
However, if you are using ClickHouse Cloud, you must first get in touch with support to enable the usage of the JSON type.
```

Reply:
Yeah lets just keep it as is and start writing some datas